components/decimal/src/provider.rs

```diff
@@ -74,3 +74,26 @@ pub struct DecimalSymbolsV1 {
     /// Zero digit for the current numbering system.
     pub zero_digit: char,
 }
+
+impl Default for DecimalSymbolsV1 {
+    fn default() -> Self {
+        Self {
+            minus_sign_affixes: AffixesV1 {
+                prefix: "-".into(),
+                suffix: "".into(),
```

Comment:
can we do this as `Option`? Seems like in most cases it'll be *either* suffix of prefix, or neighter, so it should give us a nice reduction of data size.

Reply:
I have them stored as SmallString.  Making it an Option would reduce data size in JSON, but I don't think it would have an impact on Bincode or rkyv.  I'll look into it.

Reply:
Okay, there's no impact on stack size of using `Option<SmallStr8>` instead of `SmallStr8`, and performance impact is in the noise.  I'll therefore move ahead with this change.